cockatrice/src/window_main.cpp

```diff
@@ -375,10 +378,12 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
             }
             break;
         case Response::RespClientIdRequired:
-            QMessageBox::critical(this, tr("Error"), tr("This server requires client ID's. Your client is either failing to generate an ID or you are running a modified client.\nPlease close and reopen your client to try again."));
+            QMessageBox::critical(this, tr("Error"), tr("This server requires client ID's. Your client is either failing to generate an ID or you are running a modified client.\n"
```

Comment:
It's the same tr() wrapping.^^
So it's still one string for translators, but it's easier to read in the code.
Saw it at another place in the code and liked it.